sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactFacadeEjb.java

```diff
@@ -271,9 +276,24 @@ public ContactDto saveContact(ContactDto dto, boolean handleChanges) {
 		//		}
 
 		Contact entity = fromDto(dto);
-
 		contactService.ensurePersisted(entity);
 
+		if (configFacade.isConfiguredCountry(CountryHelper.COUNTRY_CODE_SWITZERLAND) && existingContact == null) {
```

Comment:
This code should be moved to a dedicated method to keep the saveContact method as clean as possible; also, it shouldn't be restricted to Switzerland because it makes sense in general to create this task.